config/l3afd.cfg

```diff
@@ -69,5 +69,8 @@ cert-dir: /etc/l3af/certs
 cacert-filename: ca.pem
 server-crt-filename: server.crt
 server-key-filename: server.key
-cert-expiry-warning-days: 30
 # how many days before expiry you want warning
+cert-expiry-warning-days: 30
+# multiple domains seperated by comma
+# literal and regex are supported
+san-match-rules: .+l3afd.l3af.io,.*l3af.l3af.io,^l3afd.l3af.io$
```

Comment:
Should we leave this commented out and just provide the list as example? Default could be to do only CA match.